script.js

```diff
@@ -115,15 +123,17 @@ function createKeyboard (tempArray) {
             case 'Tab':
                 keyButton.classList.add('tab', 'darkerColorKey');
                 keyButton.addEventListener('click', (event) => {
```

Comment:
Maybe, I don't understand, but it is not good to have a lot of addEventListener, need to have only one on the wrapper for all buttons and in script check, which the button was clicked

Reply:
https://learn.javascript.ru/event-delegation